src/app/components/content-page/content-page-variables.scss

```diff
@@ -14,7 +12,6 @@ $card-bgColor: transparent;
 $card-border: none;
 $card-flex-direction: inherit;
```

Comment:
Will all cards within the application make use of this specific styles?

Reply:
Just all cards on **this** component. They are needed to make the cards look cleaner.  It can be changed if we have a theme.

Reply:
So according to my understanding we use variables when you know that the value will be reused a couple of time, or if it will change over time. So what I am trying to figure out is whether these variables will be used more than once? Is it needed for them to be in variables?

Reply:
I totally understand that. I assumed because we are using SCSS  we should declare variables to keep to SCSS standards. When you declare variables you are able to put them in a separate file which makes you code look a bit better. Certain variables I did reuse , but I removed all the variables now.